core/api/transport/impl/http/http_session.hpp

```diff
@@ -69,6 +69,22 @@ namespace kagome::api {
      */
     void respond(std::string_view response) override;
 
+    /**
+     * @brief method to get id of the session
+     * @return id of the session
+     */
+    SessionId id() const override {
+      return 0ull;
```

Comment:
Pretty.

Reply:
I don't need it, and i don't come up with a scenario where ```id``` will be needed for http session.